src/components/chat-transcript.scss

```diff
@@ -0,0 +1,29 @@
+@use "vars" as *;
+
+.chat-transcript__messages {
+  background: white;
+  color: $dark-gray;
+  list-style: none;
+  margin: 0;
+  max-height: 450px;
+  padding: 0;
+  overflow-y: auto;
+  text-align: left;
+
+  .chat-transcript__message {
+    margin: 0;
+    padding: 8px 10px;
+    white-space: pre-line; // maintain line breaks
+
+    &:nth-child(odd) {
+      background: $light-teal-2;
+    }
```

Comment:
Maybe it doesn't matter, but it seems like the background color should be based on the speaker, not just message order.

Reply:
Yeah, I think that's right. Lucy is working on making it that way in the debug logging branch she's currently working on.